checkmarx/tablefmt.py

```diff
@@ -0,0 +1,178 @@
+import typing
+
+import checkmarx.client
+import checkmarx.model as model
+
+import tabulate
+import textwrap
+
+
+def get_scan_info_table(
+    scan_results: typing.Iterable[model.ScanResult],
+    tablefmt: str = 'simple',
+):
+    scan_info_header = ('ScanId', 'ComponentName', 'ScanState', 'Start', 'End')
+
+    def started_on(scan_result):
+        return scan_result.scan_response.dateAndTime.startedOn if \
```

Comment:
nit: I think I would prefer not using ternary operator here for the sake of readability

Reply:
done